src/liblibc/lib.rs

```diff
@@ -227,7 +227,7 @@ pub mod types {
                 pub type rlim_t = u64;
 
                 #[repr(C)]
-                #[derive(Copy)] pub struct glob_t {
+                #[derive(Copy, Clone)] pub struct glob_t {
```

Comment:
I personally prefer to not see inherent methods and trait implementations directly on FFI types but rather on appropriate wrappers, so this is forcing the addition of a trait implementation on FFI types that want to also be `Copy`. Although others may differ on that opinion!

I do agree that inferred `derive` would be nice.